### PR TITLE
0.2.8: Fix Readymade::Controller::Serialization collection_response method raised an error.

### PR DESCRIPTION
**Pull request by mashk4**

### CODE REVIEW

File: lib/readymade/controller/serialization.rb [hunk 1]
```diff
@@ -21,15 +21,12 @@ def record_response(record, *args)
...
-          {
-            (options.delete(:root).presence || :items) => serialize_collection(paginate(collection, options), options),
-            count: collection.count
-          },
-          options[:status] || :ok
-        )
+        args.extract_options!.then do |options|
+          render_json(
+            serialize_collection(paginate(collection, options), options).merge(count: collection.count),
+            :ok
```
Comment (OrestF) [inline lib/readymade/controller/serialization.rb]: why did u remove status option?

File: lib/readymade/controller/serialization.rb [hunk 2]
```diff
@@ -21,15 +21,12 @@ def record_response(record, *args)
...
-        render_json(
-          {
-            (options.delete(:root).presence || :items) => serialize_collection(paginate(collection, options), options),
-            count: collection.count
-          },
-          options[:status] || :ok
-        )
+        args.extract_options!.then do |options|
+          render_json(
+            serialize_collection(paginate(collection, options), options).merge(count: collection.count),
```
Comment (OrestF) [inline lib/readymade/controller/serialization.rb:26]: not sure that `count` is merged in the same place - is the collection_response same as if was?

Comment (dmrAnderson) [reply lib/readymade/controller/serialization.rb:26]: @OrestF
I will double-check that

File: lib/readymade/controller/serialization.rb (hunk 1 above)
Comment (dmrAnderson) [reply lib/readymade/controller/serialization.rb]: @OrestF 
My bad, will return that back

**Comment by OrestF** [comment]
@dmrAnderson @mashk4 
what's the status of the changes?

File: lib/readymade/controller/serialization.rb (hunk 2 above)
Comment (dmrAnderson) [reply lib/readymade/controller/serialization.rb:26]: `serialize_collection(paginate(collection, options), options)` will return a hash, so we can safely merge `count`

File: lib/readymade/controller/serialization.rb (hunk 1 above)
Comment (dmrAnderson) [reply lib/readymade/controller/serialization.rb]:
```suggestion
            options[:status] || :ok
```

File: lib/readymade/controller/serialization.rb (hunk 2 above)
Comment (dmrAnderson) [reply lib/readymade/controller/serialization.rb:26]: `count` still has top level, no one breaking changes 👌

**Pull request merged**